src/utils/check.your.answers.ts

```diff
@@ -30,17 +30,15 @@ import {
 import { RoleWithinTrustType } from "../model/role.within.trust.type.model";
 
 export const getDataForReview = (req: Request, res: Response, next: NextFunction, isNoChangeJourney: boolean) => {
-  const hasAnyBosWithTrusteeNocs = checkHasAnyBosWithTrusteeNocs(getApplicationData(req.session));
+  const appData = getApplicationData(req.session);
+  const hasAnyBosWithTrusteeNocs = isNoChangeJourney ? checkEntityReviewRequiresTrusts(appData) : checkEntityRequiresTrusts(appData);
```

Comment:
are the review BOs still not popped at this point? What point do they get moved to main model in a no change journey?

Reply:
They never get moved to the main model in the no change journey. As they have not been reviewed until user click on to "pay" they are rendered directly from the update.review_ lists in the no change review page.